samza-tools/scripts/samza-sql-console.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
Sorry i don't think that is a good idea. It is good to keep the scripts that invoke the tools coupled with the tools so that if we delete the tool, we remember to delete the scripts as well.

Reply:
We have been using the cross-module coupling pattern to separate the java / scala source code from the scripts that invoking them in Samza for main production scripts. Better to be consistent on that. If all the scripts in samza-tools/bin are only used for test/demo and not for main production usage, I am OK w/ the current approach.